src/modules/fw_path_navigation/FixedwingPositionControl.hpp

```diff
@@ -689,14 +690,15 @@ class FixedwingPositionControl final : public ModuleBase<FixedwingPositionContro
 	void set_control_mode_current(const hrt_abstime &now);
 
 	/**
-	 * @brief Compensate trim throttle for air density and vehicle weight.
+	 * @brief Estimate trim throttle for air density, vehicle weight and current airspeed
```

Comment:
```suggestion
	 * @brief Estimate trim throttle from air density, vehicle weight and current airspeed
```

Reply:
no longer relevant as this part is remvoed again.